src/o365/todo/commands/list/list-remove.ts

```diff
@@ -0,0 +1,159 @@
+import commands from '../../commands';
+import request from '../../../../request';
+import GlobalOptions from '../../../../GlobalOptions';
+import {
+  CommandOption, CommandValidate, CommandError
+} from '../../../../Command';
+import GraphCommand from '../../../base/GraphCommand';
+
+const vorpal: Vorpal = require('../../../../vorpal-init');
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  id?: string;
+  name?: string;
+  confirm?: boolean;
+}
+
+class TodoListRemoveCommand extends GraphCommand {
+  public get name(): string {
+    return `${commands.LIST_REMOVE}`;
+  }
+
+  public get description(): string {
+    return 'Removes a Microsoft To Do task list';
+  }
+
+
+  public commandAction(cmd: CommandInstance, args: CommandArgs, cb: (err?: any) => void): void {
+
+    const getListId = () => {
+      if (args.options.name) {
+        // Search list by its name
+        const requestOptions: any = {
+          url: `${this.resource}/beta/me/todo/lists?$filter=displayName eq '${escape(args.options.name)}'`,
+          headers: {
+            accept: "application/json;odata.metadata=none"
+          },
+          json: true
+        };
+        return request.get(requestOptions)
```

Comment:
When attempting to delete a task list using a name that does not exist, I get an error.

![image](https://user-images.githubusercontent.com/11563347/82764979-e9f17b00-9e0a-11ea-84b1-bee54ff55fcc.png)

Reply:
I don't really get it... it's something I tested and I just double-checked, it doesn't happen on my side 
![image](https://user-images.githubusercontent.com/8345556/82765665-fd0b4780-9e18-11ea-88b3-51204b59ca1a.png)
However, I only tested it on Linux (WSL). Let me check if I can reproduce it on Windows

Reply:
Interesting, my tests have been on MacOS 🤔

Reply:
So, I double-double-checked (:P) and I don't have the issue on Windows (Node 12) nor Linux Ubuntu (Node 14). If you use --debug, What is the response of the GET ?$filter ... ?

Reply:
![image](https://user-images.githubusercontent.com/8345556/82766232-9c323e00-9e1d-11ea-8be2-c796c23f221f.png)

Reply:
Apologies @ypcode looks like this is a false alarm. After running `npm run clean && npm run build && npm link` its working as expected 👍🏻

![image](https://user-images.githubusercontent.com/11563347/82766355-50c86180-9e16-11ea-841d-4d00066e54a2.png)